examples/doc-scan/src/controllers/index.controller.js

```diff
@@ -46,10 +50,15 @@ async function createSession() {
     )
     .withRequestedTask(
       new RequestedTextExtractionTaskBuilder()
-        .withManualCheckNever()
+        .withManualCheckAlways()
```

Comment:
Do we want to change this?

Reply:
This was intentional to help demonstrate checks in the response